rest_framework_nested/runtests/urls.py

```diff
@@ -1,6 +1,6 @@
 """
 Blank URLConf just to keep runtests.py happy.
 """
-from rest_framework.compat import patterns
+from rest_framework.compat import patterns  # type: ignore[attr-defined]
```

Comment:
`rest_framework.compat.patterns` doesn't actually exist, this crashes with `ImportError`.